README.md

```diff
@@ -144,3 +144,247 @@ Generates this output:
 █████████████████████████████
 ▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀▀
 ```
+
+## PIC generation
+
+```rust
+use qrcode::render::pic;
+use qrcode::QrCode;
+
+fn main() {
+    let code = QrCode::new(b"01234567").unwrap();
+    let image = code
+        .render()
+        .min_dimensions(1, 1)
+        .dark_color(pic::Color("#800000"))
+        .light_color(pic::Color("#ffff80"))
+        .build();
+    println!("{}", image);
+}
+```
+
+Generates [PIC](https://en.wikipedia.org/wiki/PIC_(markup_language))
+output that renders as follows:
+
+```pic
+maxpswid=29;maxpsht=29;movewid=0;moveht=1;boxwid=1;boxht=1
+define p { box wid $3 ht $4 fill 1 with .nw at $1,-$2 }
+box wid maxpswid ht maxpsht with .nw at 0,0
+p(4,4,1,1)
+p(5,4,1,1)
+p(6,4,1,1)
+p(7,4,1,1)
```

Comment:
well let's not include 200 lines of code here either 😂 

link to the `test_annex_i_qr_as_pic.pic` file, or truncate the file end it with `…`.

Reply:
I assume include one _very_ long line of code isn't very helpful either 😉 hence I went ahead and truncated the code (including `…` at the end` and linked to `test_annex_i_qr_as_pic.pic` below the truncated example.